frontend/src/types.ts

```diff
@@ -562,6 +562,10 @@ export interface SystemStatus {
             share_token: string
         }
     }
+    backup: {
+        is_enabled: boolean
+        existing_bk_names: string[]
```

Comment:
what is a `bk`? Let's not abbreviate unless we really need to.

How about `existing_backups`